### PR TITLE
rpcinfo: add page

### PR DESCRIPTION
**Pull request by RH-sdavey**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/rpcinfo.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+> Makes an RPC call to an RPC server and reports what it finds.
+
+- Show full table of all RPC services registered on localhost:
+
+`rpcinfo;`
+
+- Show concise table of all RPC services registered on localhost:
+
+`rpcinfo -s localhost;`
```
Comment (pxgamer) [inline pages/linux/rpcinfo.md]: Use token syntax for `localhost`

File: pages/linux/rpcinfo.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Display table of statistics of rpcbind operations on localhost:
+
+`rpcinfo -m;`
+
+- Display list of entries of given service name (mountd) and version number (2) on localhost:
+
+`rpcinfo -l localhost mountd 2;`
```
Comment (pxgamer) [inline pages/linux/rpcinfo.md]: Use token syntax for `localhost`, `mountd` and `2`

File: pages/linux/rpcinfo.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Delete the registration for version 1 of the mountd service for all transports:
+
+`rpcinfo -d mountd 1;`
```
Comment (pxgamer) [inline pages/linux/rpcinfo.md]: Use token syntax for `mountd` and `1`

**Comment by RH-sdavey** [comment]
@pxgamer Hi, no problem, 
token syntax done, and no the ; was an addition when I getting confused about the requirements of the syntax checks,  sorry about that. 
Hope its good now.

File: pages/linux/rpcinfo.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+`rpcinfo -s {{localhost}}`
+
+- Display table of statistics of rpcbind operations on localhost:
+
+`rpcinfo -m`
+
+- Display list of entries of given service name (mountd) and version number (2) on localhost:
+
+`rpcinfo -l {{localhost}} {{mountd}} {{2}}`
```
Comment (agnivade) [inline pages/linux/rpcinfo.md]: Maybe we should change this example from localhost to some other IP, so that we have different examples.

**Pull request merged**